src/build/smart-contracts.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Smart Contracts
+---
+
+# Smart Contracts
+
+This article explains the following:
+
+* [Deployment and interaction of CW20](/build/smart-contracts.html#deployment-and-interaction-of-cw20)
+
+### Install rustup
+
+First, install [rustup](https://rustup.rs/). Once installed, make sure you have the wasm32 target. This is an important step before running any contract to ensure that you are running Rust 1.44.1+ with wasm32-unknown-unknown target installed.
+
+```
+rustup --version
+rustup default stable
+default stable
+cargo --version
+# If this is lower than 1.51.0+, update
+rustup target list --installed
+```
+
+Update all dependencies in the lockfile for cargo and rustup:
+```
+cargo update
+rustup update
+```
+
+## Deployment and interaction of CW20
+
+[CW20](https://github.com/CosmWasm/cw-plus/tree/v0.9.0/contracts/cw20-base) is equivalent to ERC20. CW-20 is similar, in some respects, to bitcoin, Litecoin, and any other cryptocurrency. CW-20 tokens are blockchain-based assets that have value and can be sent and received. The primary difference is that instead of running on their own blockchain, CW-20 tokens are issued on the CosmWasm network.
+
+This section explains how to deploy and interact with CW20 using cudos-noded.
+
+### Functions present in CW20 base standard
+
+The specification of CW20 are represented within the following:
+* [Base](https://github.com/CosmWasm/cw-plus/blob/main/packages/cw20/README.md#base)
```

Comment:
if we are fixing to the v0.9.0, this should probably be:
https://github.com/CosmWasm/cw-plus/blob/0.9.x/packages/cw20/README.md#base

note: there's no specific https://github.com/CosmWasm/cw-plus/blob/0.9.0/